include-fragment-element.js.flow

```diff
@@ -0,0 +1,12 @@
+/* @flow */
+
+declare class IncludeFragmentElement extends HTMLElement {
+  get data(): Promise<string>;
+  get src(): string;
+  set src(url: string): void;
+  fetch(request: Request): Promise<Response>;
```

Comment:
We have a few patches in the app that require this method to be public API. Ideally we can remove those and make this private.